contracts/errors/Lending.sol

```diff
@@ -247,6 +247,9 @@ error RC_RepayPartLTMin(uint256 amount, uint256 minAmount);
 /// @notice Zero address passed in where not allowed.
 error LC_ZeroAddress();
 
+/// @notice Borrower address is same as lender address.
+error LC_BorrowerAddressEqualLenderAddress();
```

Comment:
Can we rename this something shorter (shorter uses less gas) - like `LC_ReusedNote`?